e2e/test/iothub/service/RegistryManagerE2ETests.cs

```diff
@@ -42,24 +41,42 @@ public async Task RegistryManager_BadProxy_ThrowsException()
         [LoggedTestMethod]
         public async Task RegistryManager_AddAndRemoveDeviceWithScope()
         {
-            var registryManager = RegistryManager.CreateFromConnectionString(Configuration.IoTHub.ConnectionString);
-
+            string edgeId1 = _devicePrefix + Guid.NewGuid();
```

Comment:
string interpolation?

Reply:
IMO $"{_devicePrefix}{Guid.NewGuild()}" is messier than the current code. Thoughts?

Reply:
I'm ok with either. I thought the preferred way was using string interpolation.

Reply:
I think it depends on what is easiest to read. Also, generally string concatenation is okay if kept at or below 3 concats.